enacts/flex_fcst/maproom.py

```diff
@@ -330,20 +347,23 @@ def local_plots(marker_pos, start_date, lead_time):
         error_fig = pingrid.error_fig(error_msg="Grid box out of data domain")
         return error_fig, error_fig
 
-    if CONFIG["leads"] is not None and CONFIG["targets"] is not None:
-        raise Exception("I am not sure which of leads or targets to use")
-    elif CONFIG["leads"] is not None:
-        target_range = predictions.target_range_format(
-            CONFIG["leads"][lead_time],
-            lead_time,
-            pd.to_datetime(start_date),
-            CONFIG["target_period_length"],
-            CONFIG["time_units"],
-        )
-    elif CONFIG["targets"] is not None:
-        target_range = lead_time
+    if CONFIG["forecast_mu_file_pattern"] is None:
+        #target_range = predictions.target_range_formatting(fcst_mu["Ti"].compute(), fcst_mu["Tf"].compute()[0], "months")
+        target_range = "Oct-Dec"
```

Comment:
@aaron-kaplan , I have done all the possible combinations of compute, dt, data, values, indexing, I can't get it. the inputs of target_range_formatting need be dt... that's all I can figure out. Help please.